gp-includes/router.php

```diff
@@ -28,7 +28,7 @@ public function request_uri() {
 	}
 
 	public function request_method() {
-		return gp_array_get( $_SERVER, 'REQUEST_METHOD', 'GET' );
+		return wp_unslash( gp_array_get( $_SERVER, 'REQUEST_METHOD', 'GET' ) );
```

Comment:
Neither do I but as WordPress adds slashes to it, it's best to strip them out just in case.